### DISCUSSION TITLE
Only show article modified date if it was actually modified

### DISCUSSION THREAD

**Discussion by PhasecoreX** [jpanther/congo#112] [Ideas]
See title. `showDateUpdated` set to true in `params.toml` will always show the article modified date, even if one doesn't exist. I don't want all of my articles to show this date when most haven't been modified. It's set to false for now, but I would like to show the modified date for the few that have been modified.

**Comment by jpanther** [comment]
Thanks for the suggestion, I'll fix this in the next update.
